packages/nimble-components/src/accordion/specs/IxD.md

```diff
@@ -0,0 +1,193 @@
+# Nimble Accordion (IxD)
+
+## Overview
+
+The accordion component is used to present a large amount of information in a small space via progressive disclosure. Accordions make information easier to parse when not all of the information may be relevant or important to the user at the same time.
+n
+
+## Background
+
+-   [IxD recommendation for Tree, Accordion, Disclosure, Table row details (#1150)](https://github.com/orgs/ni/projects/7/views/13?pane=issue&itemId=24667320)
+-   [ViD Accordion Styles (Figma)](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-85131&mode=design)
+
+## Usage
+
+### Accordions & Disclosures
+
+Accordions are vertically stacked sets of disclosures. A disclosure is made up of a header button and a content panel. Clicking the button shows/hides the panel. The panel can have almost any type of content.
+
+Use an accordion:
+
+-   To organize large amounts of information into related panels
+-   To shorten scrolled views when not all information is equally important
+-   In views with reduced horizontal or vertical space
+
+Do not use an accordion:
+
+-   For a single panel of content
+-   To organize unrelated information into panels
+-   For selectable or navigable lists or trees
+
+Best practice is to use multiple disclosures in one accordion and avoid one disclosure in multiple accordions.
+
+<p>
+<kbd><img src="spec-images/Accordion-13.png"></kbd>
```

Comment:
nitpick: why are there `<kbd>` tags? I'd never heard of them before but learned they're for [formatting text that indicates keyboard shortcuts](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd) so it seems odd to put them around images.

I'd vote to use markdown syntax for images if we can:
```
![You can write alt text here or have blank space between the brackets](spec-images/Accordion-13.png)
```

Reply:
I was trying to make a border or box shadow around the images (because they have white backgrounds that bleed into the document background), but GitHub doesn't allow custom CSS styles in Markdown.

Reply:
My vote would be to live with GitHub's mediocre styling rather than trying to work around it.

Reply:
How do I add line breaks between text and an image? It appears to be rendering as floats instead of blocks?